go-runtime/goplugin/service.go

```diff
@@ -50,11 +50,12 @@ func (filesUpdatedEvent) updateEvent() {}
 
 // buildContext contains contextual information needed to build.
 type buildContext struct {
-	ID           string
-	Config       moduleconfig.AbsModuleConfig
-	Schema       *schema.Schema
-	Dependencies []string
-	BuildEnv     []string
+	ID              string
+	Config          moduleconfig.AbsModuleConfig
+	Schema          *schema.Schema
+	Dependencies    []string
+	BuildEnv        []string
+	AdditionalFiles []string
```

Comment:
It's not a requirement that language plugins be in Go, and in general we try to push as much as possible out of the plugins and into the engine for this reason.

The pattern is that the ModuleConfig tells the BuildEngine where to look for different parts of the build, and the BuildEngine does the work of collecting it into the schema+build. This fits that fairly well. It could be a glob, or a regex, or a map of db names to paths or something.

Reply:
I have changed this.